jovian/__init__.py

```diff
@@ -48,8 +48,16 @@ def reset(which=[]):
     _data_blocks = [(i, j) for (i, j) in _data_blocks if j not in which]
 
 
-def commit(secret=False, nb_filename=None, files=[], capture_env=True,
-           env_type='conda', notebook_id=None, create_new=None, artifacts=[], git=False, commit_msg="jovian commit"):
+def commit(secret=False,
+           nb_filename=None,
+           files=[],
+           capture_env=True,
+           env_type='conda',
+           notebook_id=None,
+           create_new=None,
+           artifacts=[],
+           do_git_commit=True,
```

Comment:
@aakashns Let `do_git_commit` be `False` by default. It can be enabled if needed and i have added a branch where it'll notify when this is set to `True` and there is git repo available.